### PR TITLE
Cdmp 2337 AWS cost alerting pipeline

### PR DESCRIPTION
**Pull request by nefischer**
pipeline linting and code reformatting for the nodejs source will be handled in a different jira ticket

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+matrix:
+  NAME:
+    - ukhomeofficedigital/cdp-ci
+  DOCKER_REPO:
+    - quay.io
```
Comment (daniel-ac-martin) [inline .drone.yml:5]: Looks like we aren't using any of these variables so I think all of the above should go.

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    when:
+      event:
+        - push
+        - tag
+
+  test_nodejs:
+    image: node:8-alpine
+    commands:
+      - apk add make
+      - make node-test
```
Comment (daniel-ac-martin) [inline .drone.yml:33]: I'd add node to the cdp-ci image.

Maybe something for the future though.

**Review by daniel-ac-martin — CHANGES_REQUESTED** [review]
Seems fine otherwise but I'm not the best person to review python.

File: .drone.yml (hunk 1 above)
Comment (nefischer) [reply .drone.yml:5]: Yes. It's actually already been removed in CDMP-2366
[.drone.yml in CDMP-2366](https://github.com/UKHomeOffice/aws-budget-alerting/blob/CDMP-2366/.drone.yml)

File: .drone.yml (hunk 2 above)
Comment (nefischer) [reply .drone.yml:33]: Yes, I think that would be a different requirement / ticket

File: .drone.yml (hunk 1 above)
Comment (nefischer) [reply .drone.yml:5]: So to clarify my comment, I suggest we leave this as it is in this PR as the following one (CDMP-2366) will take care of fixing it. The following PRs are also queued behind the latter one: CDMP-2365, CDMP-2364, CDMP-2367

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**